README.md

```diff
@@ -281,31 +281,31 @@ them in several ways:
 
 ```ruby
 # input is entire response, block can mutate it
-post_process { |hash| hash['foo'] = 'bar' }
+transform { |hash| hash['foo'] = 'bar' }
 
 # input is entire response, and response is fully replaced with block's
 # return value
-post_process { |hash| hash['foo'] } # Now only "foo"s value will be response
+transform { |hash| hash['foo'] } # Now only "foo"s value will be response
```

Comment:
`transform(replace: true)` should be here, right?..

Reply:
@zverok i mentioned in another comment of yours that I'd like to remove the `replace` option and just always return the value that should be used rather than mutate the argument to the block. Thoughts?